modules/autofill/autofill-impl/src/main/java/com/duckduckgo/autofill/ui/credential/management/viewing/AutofillManagementDisabledMode.kt

```diff
@@ -66,15 +66,18 @@ class AutofillManagementDisabledMode : Fragment() {
 
     @SuppressLint("InlinedApi")
     private fun launchDeviceAuthEnrollment() {
```

Comment:
We should probably add a suppress lint statement here since we handle the deprecation warning

```suggestion
    @Suppress("DEPRECATION")
    private fun launchDeviceAuthEnrollment() {
```